examples/adaptive_library/adaptive_scripts/TrapezoidalDuct.py

```diff
@@ -0,0 +1,184 @@
+"""
+Trapezoidal ducts
+==============================================
+Adaptive Templates script to alter rectangular ducts to trapezoidal ducts.
+"""
+# %%
+# Perform required imports
+# ------------------------
+# Import pymotorcad to access Motor-CAD.
+# Import Arc, Coordinate, Line, Region and rt_to_xy
+# to define the adaptive template geometry.
+# Import os, tempfile and shutil
+# to open and save a temporary .mot file if none is open.
```

Comment:
```suggestion
# Import PyMotoCAD to access Motor-CAD.
# Import the arc, coordinate, line, region, and ``rt_to_xy`` coordinate
# to define the adaptive template geometry.
# Import os, tempfile, and shutil
# to open and save a temporary MOT file if none is open.
```
I was uncertain how to format this line:  Import the arc, coordinate, line, region, and ``rt_to_xy`` coordinate